CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- Fix regression of Window platform extensions for macOS.
```

Comment:
I was hoping for something a bit more specific. My impression from testing this is that it fixes `with_fullsize_content_view` (...and be sure to let me know if I'm wrong on that).

Reply:
that's correct, I'll add more detail